test/deploy_changes/deploy_changes_sync/deploy_changes_sync.go

```diff
@@ -0,0 +1,67 @@
+package deploysyncchanges
+
+import (
+	"time"
+
+	"github.com/loft-sh/vcluster/pkg/platform/random"
+	"github.com/loft-sh/vcluster/test/deploy_changes/common"
+	"github.com/loft-sh/vcluster/test/framework"
+	"github.com/onsi/ginkgo/v2"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var (
+	scheme = runtime.NewScheme()
+)
+
+const (
+	pollingInterval       = time.Second * 2
+	pollingDurationLong   = time.Minute * 2
+	vclusterRepo          = "https://charts.loft.sh"
+	filePath              = "commonValues.yaml"
+	chartPath             = "../../../chart"
+	initialNsLabelKey     = "testing-ns-label"
+	initialNsLabelValue   = "testing-ns-label-value"
+	testingContainerName  = "nginx"
+	testingContainerImage = "nginxinc/nginx-unprivileged"
+)
+
+var _ = ginkgo.Describe("Deploy sync changes to vCluster", func() {
+	ginkgo.It("should deploy a vCluster using kubectl and verify sync changes ", func() {
```

Comment:
This says `deploy a vCluster` but it seems there is one already deployed. Redeploy or update would be more accurate.